src/status_im/contexts/shell/share/wallet/view.cljs

```diff
@@ -66,25 +68,58 @@
             :networks            @selected-networks
             :on-share-press      #(share-action qr-url share-title)
             :profile-picture     nil
+            :unblur-on-android?  true
             :full-name           (:name account)
             :customization-color (:color account)
             :emoji               (:emoji account)
             :on-multichain-press #(reset! wallet-type :wallet-multichain)
             :on-legacy-press     #(reset! wallet-type :wallet-legacy)
             :on-settings-press   #(open-preferences @selected-networks)}]]]))))
 
+(def wallet-qr-code-item (memoize wallet-qr-code-item-internal))
+
+(defn- indicator
+  [is-active]
+  [rn/view
+   {:style (wallet-style/indicator-wrapper-style is-active)}])
+
+(defn- indicator-list
+  [n current-index]
```

Comment:
What is `n`? Can we rename it to be more explicit?

Reply:
updated to be `indicator-count`